spec/models/log_line_spec.rb

```diff
@@ -34,6 +41,20 @@
         expect(log.serial.present?).to be_falsey
       end
     end
+    context 'w235 53214' do
+      let(:dirty_serial) { FactoryGirl.create(:serial_search, serial: 'w235 53214') }
```

Comment:
Use `SerialSearch.new(serial: 'xxxxx')` instead of factory here - firstly because not hitting the database unnecessarily is a good practice, secondly because you should be [testing that there is a before filter of some sort calling this method](https://github.com/bikeindex/bike_index/blob/master/spec/models/bike_spec.rb#L169-L173)